src/main/kotlin/lotto.kt

```diff
@@ -0,0 +1,38 @@
+class LottoGenerator {
+    fun generate(count: Int): List<List<Int>> {
+        val results = mutableListOf<List<Int>>()
+
+        repeat(count) {
+            val lotto_numbers = (1..45).shuffled().take(6).sorted()
+            results.add(lotto_numbers)
+        }
+
+        return results
+    }
```

Comment:
이 부분은 최대한 불변으로 유지하는게 어떨까요?

```suggestion
    fun generate(count: Int): List<List<Int>> {
        (1..count).map {
           (1..45).shuffled().take(6).sorted()
        }
    }
```